packages/ui/src/proposals/modals/AddNewProposal/components/SuccessModal.tsx

```diff
@@ -35,8 +35,8 @@ export const SuccessModal = ({ onClose, proposalId, proposalType, proposalTitle
         </Info>
       </ModalBody>
       <ModalFooter>
-        <ButtonGhost size="medium" onClick={openProposal}>
-          See my Proposal
+        <ButtonGhost size="medium" onClick={openProposals}>
+          Back to proposals
```

Comment:
Thank you for the PR it is exactly what is needed for the "VoteForProposal" modal. But #2838 is only targeted at this modal not at the "AddNewProposal" modal. We do want modal creators to have a direct link to the newly created modal.

Reply:
Apologies, I don't get your comment. I cannot find "See my Proposal" anywhere else in the codebase, including `VoteForProposalModal.ts`

Reply:
Yes

Reply:
Fixed